### ISSUE TITLE
[ENHANCEMENT]: Include HDR Output Transforms in the views list for Display P3

### ISSUE THREAD

**Issue by nick-shaw** [AcademySoftwareFoundation/OpenColorIO-Config-ACES#112]
Apple `kCGColorSpaceExtendedDisplayP3` supports float values above 1.0 for HDR/EDR display on capable Mac systems. Currently the views for the newly added `Display P3` display in the Studio and CG configs only include the SDR D65 and D60 sim Output Transforms. The addition of HDR Output transforms to that list would greatly simplify working in HDR for users with suitable Apple devices.

I have previously made my own [config](https://github.com/nick-shaw/ocio-configs/blob/70e1ddb6024ea984a48ade1c6340c8f60d3e7b69/studio-config-ns/studio-config-v1.0.0_aces-v1.3_ocio-v2.1_ns.ocio#L53) which does exactly this, and in my (admittedly limited) testing there appear to be no issues with mixing SDR and HDR views for the same display.

**Comment by KelSolaar** [comment]
Hey @nick-shaw,

Thanks! The starting point is to have the corresponding Output Transform on the `aces-dev` side like I did for the SDR variant. Once this is done, we can start looking at adding it to the config.

Cheers,

**Comment by nick-shaw** [comment]
Thanks @KelSolaar 

So is it the case that the config must have a 1:1 relationship with Output Transforms in `aces-dev`? I was imagining that given the separation of views and displays in OCIO, it would only be necessary to add to the `Display P3` views list, to indicate that (on suitable devices) it was capable of showing HDR as well as SDR view transforms. Probably only `ACES 1.1 - HDR Video (1000 nits & P3 lim)`.

**Comment by KelSolaar** [comment]
Yes, we derive the *Reference* transforms to build from `aces-dev` and use a spreadsheet that links a given `ACEStransformID` to the relevant OCIO `BuiltinTransform` and `Display Colorspace`: https://docs.google.com/spreadsheets/d/1SXPt-USy3HlV2G2qAvh9zit6ZCINDOlfKT07yXJdWLg/edit#gid=0

**Comment by nick-shaw** [comment]
[This commit](https://github.com/ampas/aces-dev/compare/dev...nick-shaw:aces-dev:Display_P3_HDR) adds Display P3 1000 nit forward and inverse Output Transforms. But it is not clear to me what the file names and transform IDs should be.

Thoughts?

Tagging @scottdyer and @aforsythe.

**Comment by doug-walker** [comment]
This would require a separate display rather than overloading the existing SDR Display P3. Apple has an extended Display P3 color space, so perhaps that could inform the name. It needs to be clear whether a display is either an SDR or HDR target.

**Comment by doug-walker** [comment]
The ACES 2 configs have "Display P3 HDR - Display", so I will close this as completed.
